src/quick-lint-js/diag/diagnostic-types.h

```diff
@@ -2976,6 +2976,15 @@
       diag_unexpected_colon_after_generic_definition, "E0331",                  \
       diagnostic_severity::error, { source_code_span colon; },                  \
       MESSAGE(QLJS_TRANSLATABLE("':' should be 'extends' instead"), colon))     \
+                                                                                \
+  QLJS_DIAG_TYPE(                                                               \
+      diag_access_specifier_must_proceed_other_modifiers, "E0369",              \
+      diagnostic_severity::error, {                                             \
+        source_code_span specifier;                                             \
+        source_code_span modifier;                                              \
```

Comment:
Nit: I think these are confusing names. I don't know what the difference between a specifier and modifier is.

Perhaps 'second_modifier' and 'first_modifier' are better names.

Reply:
Yep, specifier and modifier wasn't clear enough. Changed to 'second_modifier' and 'first_modifier'